builder/steps/gen-dockerfile/contents/src/fsview.ts

```diff
@@ -102,17 +102,17 @@ export class FsView implements Reader, Writer, Locator {
   }
 
   /** @inheritdoc */
-  async read(path: string) {
-    return pify(fs.readFile)(this.resolvePath(path), 'utf8');
+  async read(p: string) {
```

Comment:
What's the rationale for making the variable name terse?

Reply:
The local name `path` shadows the `path` import, and I couldn't think of a better name.